### ISSUE TITLE
Heroku feedback gives a 404 Error for Github

### ISSUE THREAD

**Issue by harraton** [mozilla/firefox_voice_community#233]
"Also, I just submitted a bug via the Heroku feedback app, and it had a link to click to see my issue on GitHub. That link went to a 404 error."

https://github.com/mozilla/firefox_voice_community/issues/231

**Comment by MichaelKohler** [comment]
The main cause is that the bot is flagged. This could be due to what @awallin mentioned. I've written two mails to GitHub support but they are nicely ignoring me.

For reference: it's Ticket ID 526302

**Comment by MichaelKohler** [comment]
We probably could point people directly to https://github.com/mozilla/firefox_voice_community/issues/new/choose in the meantime?

**Comment by awallin** [comment]
@MichaelKohler Is it possible to redirect from the form to the page you linked above? We link to the form from a number of place including in emails (previously sent), in product (would require release of new build) and others.

**Comment by MichaelKohler** [comment]
Agreed, that's the best way for now. https://firefox-voice-feedback.herokuapp.com/ is now redirecting to https://github.com/mozilla/firefox_voice_community/issues/new/choose . The downside of this is that users need a GitHub account, but at least we can get feedback somewhere.

**Comment by MichaelKohler** [comment]
I'll leave this open until I get a response from GitHub. I still want these issues back!

**Comment by awallin** [comment]
Thanks! Yes, we definitely want the issues back.
